frontend/src/api/k8s/dashboardConfig.ts

```diff
@@ -43,3 +46,25 @@ export const patchDashboardConfigTemplateOrder = (
     }
     return dashboardConfig.spec?.templateOrder;
   });
+
+export const patchDashboardConfigTemplateDisablement = (
+  templateDisablement: string[],
+  ns: string,
+): Promise<string[]> =>
+  k8sPatchResource<DashboardConfigKind>({
+    model: ODHDashboardConfigModel,
+    queryOptions: { name: DASHBOARD_CONFIG, ns },
+    patches: [
+      {
+        op: 'replace',
+        path: '/spec/templateDisablement',
+        value: templateDisablement,
+      },
+    ],
+  }).then((dashboardConfig) => {
+    // Patch doesn't return an error if the attribute is disabled, it just return the object without changes
+    if (dashboardConfig.spec?.templateDisablement === undefined) {
+      throw new Error('Template order is not configured');
```

Comment:
The text here is not updated

Reply:
Agh my bad, I miscopied and misread that one, thanks @DaoDaoNoCode